### PR TITLE
add taraxa chains to chainlist

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

File: _data/icons/taraxa.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "url": "ipfs://QmQhdktNyBeXmCaVuQpi1B4yXheSUKrJA17L4wpECKzG5D",
+  "width": 512,
+  "height": 512,
```
Comment (ligi) [inline _data/icons/taraxa.json:4]: size seems to be wrong

identify QmQhdktNyBeXmCaVuQpi1B4yXheSUKrJA17L4wpECKzG5D         19:30:11
QmQhdktNyBeXmCaVuQpi1B4yXheSUKrJA17L4wpECKzG5D PNG 310x310 310x310+0+0 8-bit sRGB 28094B 0.000u 0:00.000

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/taraxa.json (hunk 1 above)
Comment (VargaElod23) [reply _data/icons/taraxa.json:4]: Hmm weird, our designer named it 512. I'm addressing it now:)

**Review by ligi — CHANGES_REQUESTED** [review]
the explorer does not really seem to support EIP3091 when it comes to blocks

**Comment by VargaElod23** [comment]
@ligi not in the current version but we're weeks away from releasing the next version, which, besides having additional features, supports EIP-2091 too. However, in this case I can remove it until.

**Comment by VargaElod23** [comment]
@ligi updated without EIP-3091

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `TARA` more than once

**Comment by VargaElod23** [comment]
@ligi fixed

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by VargaElod23** [comment]
@ligi formatted per your request

**Review by ligi — APPROVED** [review]

**Pull request merged**